bin/input_masterlist_ml.yaml

```diff
@@ -0,0 +1,130 @@
+defaultTimeOut: 500 #customize the default timeout threshold
+
+## To inject faults into all the layers of the model:
+## LayerNo: '0' indicates all layers of the type mentioned in LayerName; For layerName, you can specify 'all' to inject fault in all the layers
+compileOption:
+    instSelMethod:
+      - customInstselector:
+          include:
+            - CustomTensorOperator
+          options:
+            -layerNo=0 
```

Comment:
Does this option need to be explicitly included even if all layers are going to be injected? If so, we should say this in the comments. If not, we should say that it is set by default.

Reply:
I will check this and make add a comment accordingly

Reply:
The option is not set by default. I have added this in the comments.